src/Chapter1.hs

```diff
@@ -619,8 +637,12 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n
+  | num < 10 = num
+  | otherwise = num `rem` 10 + (num `div` 10) `rem` 10
```

Comment:
Yeap, great!
Note, that `mod x 10` (or rem that you use here) is the same as `lastDigit` function from the previous task 🙂

Reply:
Oh! Nice that you brought this up! Using lastDigit here will make the solution more elegant :smile: